mm2src/mm2_main/tests/mm2_tests/mm2_tests_inner.rs

```diff
@@ -871,21 +872,62 @@ async fn trade_base_rel_electrum(
         assert_eq!(0, bob_orderbook.asks.len(), "{} {} asks must be empty", base, rel);
     }
 
-    #[cfg(target_arch = "wasm32")]
-    {
-        const STOP_TIMEOUT_MS: u64 = 1000;
+    (mm_bob, mm_alice, uuids[0].clone())
+}
 
-        mm_bob.stop_and_wait_for_ctx_is_dropped(STOP_TIMEOUT_MS).await.unwrap();
-        mm_alice
-            .stop_and_wait_for_ctx_is_dropped(STOP_TIMEOUT_MS)
-            .await
-            .unwrap();
-    }
-    #[cfg(not(target_arch = "wasm32"))]
+#[test]
+#[cfg(not(target_arch = "wasm32"))]
+fn trade_test_electrum_and_eth_coins_validate_pubkeys() {
```

Comment:
I now see that there is no way / API to get the pubkey from the sqlite db. `stats_swap_status` uses the saved swap files. You can delete this test since it doesn't do what is required.

Reply:
But I think this test still verifies the pub keys... @shamardy